geomstats/geometry/special_euclidean.py

```diff
@@ -34,54 +35,155 @@
                         - 1. / 362880.]
 
 
-class SpecialEuclidean(LieGroup):
-    """Class for the special euclidean group SE(n).
+class _SpecialEuclideanMatrix(GeneralLinear, LieGroup):
+    """Class for special orthogonal groups.
 
-    i.e. the Lie group of rigid transformations. Elements of SE(n) can either
-    be represented as vectors (in 3d) or as matrices in general. The matrix
-    representation corresponds to homogeneous coordinates.
+    Parameters
+    ----------
+    n : int
+        Integer representing the shape of the matrices: n x n.
```

Comment:
actually matrices are of shape n + 1, n is the dim of the space on which this group acts by isometries